fix(side-panel-description): Fixed Side panel for people displays description

aliraza556 opened the pull request:
### Problem:
A bug was identified in the person card display component where the word `description` is incorrectly shown when no description is provided for a person. This is misleading and does not align with intuitive user interface practices.

### Expected Behavior:
The ideal behavior is for the description field to remain blank when no description is provided. This ensures a cleaner and more professional appearance of the UI, avoiding any confusion for the users.

## Issue ticket number and link:
- **Ticket Number:** [ 1207 ]
- **Link:** [ https://github.com/stakwork/sphinx-tribes/issues/1207 ]

### Solution:
The proposed solution involves modifying the condition that checks for and renders the `description` property. The updated condition now checks if the `description` property exists and is not equal to the placeholder text 'description'. Only when these conditions are met, the description is rendered. This approach effectively handles cases where the description might be absent or not applicable.

### Changes:
 - Updated the conditional rendering logic in the `Person` component for both the small and desktop card versions.
 - Ensured that the placeholder text `description` is not displayed when no description is provided.
 - Maintained existing functionality and styling for cases where the description is present.

### Evidence:
 Please see the attached image as evidence.
- Image#1: [Link](https://i.imgur.com/djNQhIy.png)
- Image#2: [Link](https://i.imgur.com/FV6JQow.png)
- Image#3: [Link](https://i.imgur.com/XuKo9cX.png)

### Testing:
- Manually tested the person card component with various states: with description, without description, and with the placeholder text `description`.
- Manual testing across different devices and screen resolutions.
- Verified that no other functionalities are impacted by this change.

### Notes:
Please review the changes and merge this PR if everything is in order. This fix is essential for the upcoming release, and I look forward to any feedback you might have.

kevkevinpal commented on frontend/app/src/pages/people/Person.tsx:
```diff
@@ -190,7 +190,7 @@ export default function Person(props: PersonProps) {
             <DTitle>{owner_alias}</DTitle>
-            {description && (
+            {description && description !== 'description' &&(
               <DDescription>{description === 'description' ? '' : description}</DDescription>
```
looks like this already has a check on the line below, does that check not work?

aliraza556 replied:
Apologies @kevkevinpal, I realized I committed to the wrong fork. Give me 10 minutes, and I'll have the correct version pushed. Thanks for pointing it out!

aliraza556 commented:
@kevkevinpal, please review the changes. Let me know if anything else is needed.

aliraza556 commented:
Hi @kevkevinpal 
I hope you're having a wonderful New Year's celebration! Just wanted to drop a quick note to let you know that I've addressed the issue and updated the code in the correct repository.

Please take a look at your earliest convenience. There's no rush, considering the holiday. Let me know if there's anything else that needs attention when you're back.

Thank you and Happy New Year!

aliraza556 commented:
Hi @kevkevinpal,
Fantastic! I'm glad to hear the updates look good on your end. I appreciate your initiative to adjust the description in the database too. That sounds like a prudent step for now.
